UPGRADING

```diff
@@ -58,7 +58,7 @@ Core:
   . Array destructuring now supports reference assignments using the syntax
     [&$a, [$b, &$c]] = $d. The same is also supported for list().
     (RFC: https://wiki.php.net/rfc/list_reference_assignment)
-  . instancof now allows literal as the first operand,
+  . instanceof now allows literal as the first operand,
     in this case the result always be FALSE.
```

Comment:
s/literal/literals and s/in this case/in which case and s/always be/is always. Quite an amazing concentration of typos in two lines ^^